### PR TITLE
Add load balancer exercises

### PR DESCRIPTION
**Pull request by href**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,17 @@ Even though we offer a number of "off the shelf" images (like Ubuntu, Debian, or
 * `05.03` - [Manually Create a Custom Image (Advanced)](./exercises/05-custom-images.md#0503---manually-create-a-custom-image-advanced)
 * `05.04` - [Automatically Build a Custom Image (Advanced)](./exercises/05-custom-images.md#0504---automatically-build-a-custom-image-advanced)
 
+### `06` [Load Balancers](./exercises/06-load-balancers.md)
+
+Modern deployments in general, and Kubernetes deployments in particluar, often employ load balancers to distribute load, improve uptime, and to increase security.
```
Comment (subsecond) [inline README.md]: Please get rid of the last "to".

File: exercises/06-load-balancers.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# `06` Load Balancers
+
+Modern deployments in general, and Kubernetes deployments in particluar, often employ load balancers to distribute load, improve uptime, and to increase security.
```
Comment (subsecond) [inline exercises/06-load-balancers.md]: Same here: Please drop the last "to".

File: exercises/06-load-balancers.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+* [Cloudscale Cloud Controller Manager](https://github.com/cloudscale-ch/cloudscale-cloud-controller-manager)
+* [K8test](https://github.com/cloudscale-ch/k8test)
+* [Blog Post: K8s Cloud Controller Manager for cloudscale](https://www.cloudscale.ch/en/news/2024/03/05/cloud-controller-manager)
+* [Blog Post: Load Balancer "as a Service"](https://www.cloudscale.ch/en/news/2023/04/28/load-balancer-as-a-service)
+
+## 🎓 Exercises
+
+### `06.01` - Load Balancer Components
+
+Load balancers include components, read the documentation to understand what these end-points manage:
```
Comment (subsecond) [inline exercises/06-load-balancers.md]: Please change to "Load balancers consist of various components. Read..."
If "load balancer" does not have a dash, then "end points" does not have one either.

File: exercises/06-load-balancers.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- Launch VMs and configure their outbound interface.
+- Include a set of servers in the load-balanced backend. 
+
+### `06.02` - Load Balancer
+
+Create two servers, and have them return an identifier when queried via HTTP (e.g., by installing `nginx` and changing the default website).
+
+The servers must be in a shared private subnet. It is okay to include a public interface to ease configuration. In a real scenario you might use a bastion host instead, and access the servers from there.
+
+Using your favourite approach (Terraform, Ansible, REST), create a load-balancer that includes these servers as pool members. When queried via HTTP, the load-balancer should actually return answers from both servers (round-robin).
```
Comment (subsecond) [inline exercises/06-load-balancers.md]: Inconsistency: "load-balancer" without a dash everywhere else.

**Review by subsecond — CHANGES_REQUESTED** [review]
Sorry for the delay. Great work!

Some minor text corrections. Other than that: Ship it!

File: exercises/06-load-balancers.md (hunk 4 above)
Comment (href) [reply exercises/06-load-balancers.md]: Done.

**Pull request merged**